common/tlo-parsing/tlo-parsing.cmake

```diff
@@ -25,8 +25,8 @@ set_target_properties(tlo_parsing_static PROPERTIES
 
 install(TARGETS tlo_parsing_static
         COMPONENT TLO_PARSING_DEV
-        ARCHIVE DESTINATION lib
-        PUBLIC_HEADER DESTINATION usr/include/tlo-parsing)
```

Comment:
Let's leave current behavior on debian-like systems.
```
usr/${CMAKE_INSTALL_INCLUDEDIR}/tlo-parsing
```

Reply:
I don't quite understand the reason for this decision. By specifying an absolute path, we are sure that the headers will be installed in `/usr/${CMAKE_INSTALL_INCLUDEDIR}/*`.

If we use a relative path, problems may arise if `CMAKE_INSTALL_PREFIX` is not defaulted. Some build systems replace it with cmake options.  (I'm talking specifically about [rpmbuild](https://docs.fedoraproject.org/en-US/packaging-guidelines/CMake/#_available_macros) macros, but perhaps others systems may also have similar behavior).

In this case, duplication will occur, and libraries will begin to be installed in `/usr/usr`. This doesn't seem like desirable behavior.

UPD: Now during the check I discovered that in the `install_dir/usr/usr/share/vkontakte`  "usr" directory is duplicated when `CMAKE_INSTALL_PREFIX` is set to `/usr`.

It seems to me rational to replace this part with an absolute path too. What do you think?

Reply:
OK, you're right